### PR TITLE
add config argument for removal efficiency calculation method

### PR DESCRIPTION
**Pull request by ElmiraShamlou**
## Fixes/Addresses/Summary/Motivation:

This pull request adds a new `config` argument for removal efficiency calculation using concentration-based and load-based approaches. 

## Changes proposed in this PR:
- Config argument added
- Case studies updated according to concentration_based removal efficiency
- Tests updated

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my
contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.md file
   at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has
   rights to intellectual property that includes these contributions, I represent that I have
   received permission to make contributions and grant the required license on behalf of that
   employer.

### CODE REVIEW

File: pareto/strategic_water_management/run_strategic_model.py [hunk 1]
```diff
@@ -137,7 +138,8 @@
         "pipeline_cost": PipelineCost.distance_based,
         "pipeline_capacity": PipelineCapacity.input,
         "node_capacity": True,
-        "water_quality": WaterQuality.false,
+        "water_quality": WaterQuality.post_process,
```
Comment (tarnold17) [inline pareto/strategic_water_management/run_strategic_model.py:141]: What is your rationale for changing the default for `"water_quality"` from `false` to `post_process`?

**Comment by tarnold17** [comment]
At first glance, this looks great to me! A few quick comments:

1) Documentation updates should be a part of this PR - i.e., the documentation should explain both of the options for the removal efficiency calculations.
2) I think you should add a test to `test_strategic_model.py` to test the strategic model with the non-default option for `removal_efficiency_method`.

File: pareto/strategic_water_management/run_strategic_model.py (hunk 1 above)
Comment (melody-shellman) [reply pareto/strategic_water_management/run_strategic_model.py:141]: Good question. I would think this should stay the same.

Comment (ElmiraShamlou) [reply pareto/strategic_water_management/run_strategic_model.py]: I made a change to test the treatment removal efficiency argument, but I forgot to revert the change before pushing. I updated it again to set the argument back to false.

**Review by melody-shellman — COMMENTED** [review]
Minor comments! Tested with both configs options with toy case study and works as expected.

**Review by tarnold17 — APPROVED** [review]
Looks good to me - great work!

**Review by melody-shellman — APPROVED** [review]
Looks great! Thanks for adding extra notes/docs to the input sheet and run file. 
I confirmed expected results with post-process on toy case study.

**Pull request merged**